### PR TITLE
added release 0.12 and k8s 1.25 and 1.26 to table

### PR DESCRIPTION
**Pull request by KlavsKlavsen**
add release 0.12 and k8s 1.25 and 1.26 to table in readme

### CODE REVIEW

**Comment by simonpasquier** [comment]
:wave: thanks for the PR! The doc linter isn't happy apparently.

**Comment by PhilipGough** [comment]
I think this one should be closed in favour of the pre-existing #2000

**Comment by PhilipGough** [comment]
@KlavsKlavsen sorry, I did not look closely enough, we can add this, but we should remove the oldest version from the matrix too so feel free to do so

**Comment by KlavsKlavsen** [comment]
Lint fixes, per codium lint comments - and removed k8s 1.20 from list.

**Comment by KlavsKlavsen** [comment]
Appearently that did not make your linter happy..  :(
I simply cannot see the difference in the "formatted" vs. original line it is complaining about - its really unhelpful output.
Any tips on what it could be - that codium standard markdown linter doesn't complain about?

**Comment by sylvainfaivre** [comment]
Hi, updating the compatibility matrix would be good, as k8s 1.26 and 1.27 have been released. The thing is, the CI logs for this PR have expired, so how could it be merged ?

File: README.md [hunk 1]
```diff
@@ -40,13 +40,14 @@ no effect, but is still deployed.
...
+|-------------------------------------------------------------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|
+| [`release-0.8`](https://github.com/prometheus-operator/kube-prometheus/tree/release-0.8)   | ✔               | ✗               | ✗               | ✗               | ✗               | ✗               |
+| [`release-0.9`](https://github.com/prometheus-operator/kube-prometheus/tree/release-0.9)   | ✔               | ✔               | ✗               | ✗               | ✗               | ✗               |
+| [`release-0.10`](https://github.com/prometheus-operator/kube-prometheus/tree/release-0.10) | ✗               | ✔               | ✔               | ✗               | ✗               | ✗               |
+| [`release-0.11`](https://github.com/prometheus-operator/kube-prometheus/tree/release-0.11) | ✗               | ✗               | ✔               | ✔               | ✗               | ✗               |
+| [`release-0.12`](https://github.com/prometheus-operator/kube-prometheus/tree/release-0.11) | ✗               | ✗               | ✗               | ✔               | ✔               | ✔               |
```
Comment (mkiejda) [inline README.md:49]:
```suggestion
| [`release-0.12`](https://github.com/prometheus-operator/kube-prometheus/tree/release-0.12) | ✗               | ✗               | ✗               | ✔               | ✔               | ✔               |
```

**Review by mkiejda — CHANGES_REQUESTED** [review]

**Comment by philipgough** [comment]
Closing in favour of #2121

**Pull request closed without merging**